### ISSUE TITLE
Builder: adopt Pilcrow

### ISSUE THREAD

**Issue by m4rc1e** [googlefonts/gftools#349]
It may be nice to adopt and improve Simon's [Pilcrow](https://github.com/simoncozens/pilcrow) for the builder. Writing yml config files is for devops folks, not designers. The added benefit of having a UI is there's less chance of bad inputs.

**Comment by m4rc1e** [comment]
This is just a thought, not something we should necessarily implement.

**Comment by simoncozens** [comment]
Definitely. The builder was written to create a programmable GF-style font build, precisely so that it could be added to Pilcrow. I suggest

- [ ] Replacing the current Pilcrow font-building bit with calls to gftoools.builder
- [ ] Pilcrow also writes out a builder config file which can be added to the repository once the designer is happy with the designspace etc, to enable reproducible unattended builds.
